model_compression_toolkit/common/framework_implementation.py

```diff
@@ -66,6 +108,23 @@ def model_builder(self,
         raise NotImplemented(f'{self.__class__.__name__} have to implement the '
                              f'framework\'s model_builder method.')
 
+    @abstractmethod
+    def run_model_inference(self,
+                            model: Any,
+                            input_list: List[Any]) -> Tuple[Any]:
```

Comment:
This function should return a Tuple/List of NumPy arrays no?

Reply:
No, Tensorflow returns numpy, Pytorch returns a torch.Tensor